dev/ToastNotifications/ToastProgressDataABI.h

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include <frameworkUdk/ToastNotificationsRT.h>
+
+struct ToastProgressDataABI : winrt::implements<ToastProgressDataABI, ::ABI::Microsoft::Internal::ToastNotifications::IToastProgressData>
```

Comment:
Probably after @hulumane does the renaming work, this class can just be named `ToastProgressData` 😊.

Reply:
yep. Please file a bug Sharath. We can address after feature complete.

Reply:
https://github.com/microsoft/WindowsAppSDK/issues/2067